### PR TITLE
[Janitor] Remove trailing spaces

### PR DESCRIPTION
**Pull request by deckardlo**
Remove trailing spaces

Signed-off-by: Andre Loureiro <loureiro@work.co>

### CODE REVIEW

File: sudocode1 [hunk 1]
```diff
@@ -6,12 +6,12 @@ Variable def sudo1V1
 Variable def sudo1V2
 
 func methodSudoA() {
-	return sudo1V1 + sudo1V2
```
Comment (VitorMakoto) [inline sudocode1:9]: Pq vc tirou isso?

**Review by VitorMakoto — CHANGES_REQUESTED** [review]

File: sudocode1 (hunk 1 above)
Comment (deckardlo) [reply sudocode1:9]: pq tah errado

**Review by VitorMakoto — APPROVED** [review]

File: sudocode2 [hunk 2]
```diff
@@ -6,29 +6,29 @@ Variable def sudo2V1
 Variable def sudo2V2
 
 func methodSudo2A() {
-	return sudo2V1 * sudo2V2
+    return sudo2V1 * sudo2V2
```
Comment (VitorMakoto) [inline sudocode2]: Update this with the `/` operartor

**Review by VitorMakoto — CHANGES_REQUESTED** [review]

**Review by VitorMakoto — APPROVED** [review]

**Comment by VitorMakoto** [comment]
LGTM

**Pull request merged**